templates/definition/charger/ocpp-homecharge.yaml

```diff
@@ -0,0 +1,21 @@
+template: homecharge
+products:
+  - brand: Homecharge
+    description:
+      generic: Homecharger HC11L, HC22L
+requirements:
+  description:
+    de: |
+      Die Verwendung mit evcc erfordert einen eingebauten Stromzähler (Ausführungen HC11L/HC22L Energy oder Profi).
+      Die OCPP-Konfiguration erfolgt über den EFR-SECC-Ladecontroller über die URL http://<<boxip>>/secc
+      Den Zugang erfragen Sie bitte beim Hersteller EFR (www.efr.de) oder Ihrem Händler.
+    en: |
+      The charger must be equipped with a built-in meter (models HC11L/HC22L Energy or Profi).
+      For the OCPP configuration, you need to access the EFR-SECC charge controller at http://<<boxip>>/secc
+      For login credentials, ask your dealer or the vendor EFR (www.efr.de).
+params:
+  - preset: ocpp
+render: |
+  {{ include "ocpp" . }}
+  getconfiguration: false
```

Comment:
@tvand Könntest du mir nochmal sagen warum hier `getconfiguration: false` genutzt wird obwohl die Box laut dem Logauschnitt unter https://github.com/evcc-io/evcc/issues/10339 korrekt antwortet?

> ```
> [ocpp  ] TRACE 2023/09/02 12:49:19 sent JSON message to Wall-E: [2,"4010434209","GetConfiguration",{}]
> [ocpp  ] TRACE 2023/09/02 12:49:19 received JSON message from Wall-E: [3,"4010434209",{"configurationKey":[{"key":"AuthorizeRemoteTxRequests","readonly":"true","value":"false"},{"key":"ChargeProfileMaxStackLevel","readonly":"true","value":5},{"key":"ChargingScheduleAllowedChargingRateUnit","readonly":"true","value":"W"},{"key":"ChargingScheduleMaxPeriods","readonly":"true","value":20},{"key":"ClockAlignedDataInterval","readonly":"false","value":900},{"key":"ConnectionTimeOut","readonly":"false","value":300},{"key":"ConnectorPhaseRotation","readonly":"false","value":"unknown"},{"key":"GetConfigurationMaxKeys","readonly":"true","value":10},{"key":"HeartbeatInterval","readonly":"false","value":60},{"key":"LocalAuthListEnabled","readonly":"false","value":"false"},{"key":"LocalAuthListMaxLength","readonly":"true","value":20},{"key":"LocalAuthorizeOffline","readonly":"false","value":"true"},{"key":"LocalPreAuthorize","readonly":"false","value":"true"},{"key":"MaxChargingProfilesInstalled","readonly":"true","value":15},{"key":"MeterValueSampleInterval","readonly":"false","value":300},{"key":"MeterValuesAlignedData","readonly":"false","value":"Energy.Active.Import.Register,Power.Active.Import"},{"key":"MeterValuesSampledData","readonly":"false","value":"Energy.Active.Import.Register,Power.Active.Import"},{"key":"NumberOfConnectors","readonly":"true","value":1},{"key":"NumberOfResetRetries","readonly":"false","value":0},{"key":"SendLocalListMaxLength","readonly":"true","value":20},{"key":"StopTransactionOnEVSideDisconnect","readonly":"false","value":"true"},{"key":"StopTransactionOnInvalidId","readonly":"false","value":"false"},{"key":"StopTxnAlignedData","readonly":"false","value":"Energy.Active.Import.Register,Power.Active.Import"},{"key":"StopTxnSampledData","readonly":"false","value":"Energy.Active.Import.Register,Power.Active.Import"},{"key":"SupportedFeatureProfiles","readonly":"true","value":"Core,LocalAuthListManagement,SmartCharging"},{"key":"TransactionMessageAttempts","readonly":"false","value":3},{"key":"TransactionMessageRetryInterval","readonly":"false","value":180},{"key":"UnlockConnectorOnEVSideDisconnect","readonly":"false","value":"false"},{"key":"ckchargepowermax","readonly":"true","value":11000}]}]
> ```

Reply:
Wie wir in https://github.com/evcc-io/evcc/issues/10339 ausführlich diskutiert haben, tut sie das eben nicht. Siehe auch den Kommentar von @andig hier: https://github.com/evcc-io/evcc/issues/10339#issuecomment-1763873292

Reply:
Ach ja, danke!
Und bis heute wahrscheinlich kein Firmware-Fix des Herstellers, oder? :/

Reply:
Ich habe das nicht weiter verfolgt und bin zwischenzeitlich auf einen Fronius Wattpiloten umgestiegen, der _viel_ besser mit meiner PV harmoniert.